components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -99,6 +102,33 @@ export const LockScreen = () => {
 
   const isAndroid = loadTimeData.getBoolean('isAndroid') || false
 
+  if (isAndroid) {
+    // Methods
+    const onDoubleTap = (e: React.MouseEvent<HTMLButtonElement>) => {
+      if (e.detail === 2) {
+        getWalletAPIProxy().pageHandler?.unlockWalletUI()
+      }
+    }
```

Comment:
nit: can we move this to the existing `// methods` section?

Reply:
Moved in https://github.com/brave/brave-core/pull/24992/commits/7bb8853179aa94459eddd166de6aecfc376cb0ce